substrate/frame/nomination-pools/src/migration.rs

```diff
@@ -54,11 +54,44 @@ pub mod versioned {
 		crate::pallet::Pallet<T>,
 		<T as frame_system::Config>::DbWeight,
 	>;
+
+	/// On this version of the code migration to V7 will not succeed unless migration to v8 was done
```

Comment:
Please just make a comment that we can change order of migrations since they touch different storage maps

Reply:
added a comment - a little changed. They use the same storage map but are independent